### ISSUE TITLE
Pip and other aliases in Mac Catalina / ZSH

### ISSUE THREAD

**Issue by payamazadi** [realpython/python-guide#1018]
Mac Catalina uses ZSH by default. Which is what you'll get if you buy a new Mac.
If you follow the guide, https://docs.python-guide.org/starting/install3/osx/#pipenv-virtual-environments, at the end, if you just run "pip" in your terminal, it will not be found.

After I changed the default terminal to bash, and reinstalled python with brew, then running "pip --version" gives pip version 3 as expected.

**Comment by sandrogisler** [comment]
Note that under the newest macOS, python is already installed.
Fr that general problem, I opened
https://github.com/realpython/python-guide/issues/1032

**Comment by brennanyoung** [comment]
Yes, I followed the instructions on **Installing Python 3 on Mac OS X** closely on my new M3 running Sonoma and it simply didn't work. The doc talks about the bash shell, but the default shell has been zsh for quite a while. The homebrew install instructions do not work, This doc desperately needs an update. 

I am not a python novice. **I have been doing python development on Windows for years**, and it is so much easier to get up and running on that system. I can also remember that python used to "just work" on Mac, but those days are over, and these Installation docs are not much help.

After a lot of faffing about, visiting another page about homebrew I finally got something working with homebrew, so now python3 is installed, and I enter the python environment when I type `python3` but the aliases `python` and `pip` give me "command not found". I do not know where to go from here. I thought homebrew was supposed to take care of all that stuff.

I am not looking for tech support, I just want to point out that the current instructions are simply not adequate for a wide range of Mac installations. If someone like me, with python development experience, sitting at a new Mac can't actually install python following your instructions, well, Houston, we have a problem.
